packages/components/button-bar/src/button-bar.stories.ts

```diff
@@ -20,31 +20,73 @@ export default {
       options: ['start', 'center', 'end', 'space-between']
     }
   },
+  parameters: {
+    viewport: {
+      defaultViewport: 'default'
```

Comment:
Do you need to set this, since it's the default anyway?

Reply:
yes, when you switch back from mobile to default you want this, otherwise is stays on the mobile viewport